.github/workflows/master.yml

```diff
@@ -153,11 +153,11 @@ jobs:
       - name: Build and push
         uses: docker/build-push-action@v6
         with:
-          context: java/appdynamics
+          context: java
           platforms: linux/amd64,linux/arm64
-          build-args: base_image=eclipse-temurin:${{matrix.version}}-jre
+          build-args: base_image=eclipse-temurin:${{matrix.version}}-jre-alpine
           pull: true
           push: ${{ github.ref_name == 'master' }}
-          tags: ghcr.io/${{ github.repository }}/temurin:${{matrix.version}}-appdynamics
+          tags: ghcr.io/${{ github.repository }}/temurin:${{matrix.version}}-alpine
```

Comment:
Får skape en ny tag for alpine, bytter du denne så slutter vi oppdatere image for de som er kvar på -appdynamics taggen.


Men om har man tid å bytte tag har man kanskje tid å bytte til distroless? 😁